lib/dojos.js

```diff
@@ -0,0 +1,190 @@
+  }, {
+    method: 'GET',
+    path: options.basePath + '/dojos/dojo-lead/{id}',
+    handler: handlers.actHandlerNeedsUser('load_dojo_lead', 'id')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/dojos/setup-steps',
+    handler: handlers.actHandlerNeedsUser('load_setup_dojo_steps')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/users',
+    handler: handlers.actHandler('load_usersdojos')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/search-dojo-leads',
+    handler: handlers.actHandler('search_dojo_leads')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/dojos/uncompleted',
+    handler: handlers.actHandler('uncompleted_dojos')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/load-dojo-users',
+    handler: handlers.actHandler('load_dojo_users')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/generate-user-invite-token',
+    handler: handlers.actHandler('generate_user_invite_token')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/accept-user-invite',
+    handler: handlers.actHandler('accept_user_invite')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/request-user-invite',
+    handler: handlers.actHandler('request_user_invite')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/accept-user-request',
+    handler: handlers.actHandler('accept_user_request')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/dojos/dojos-for-user/{id}',
+    handler: handlers.actHandler('dojos_for_user', 'id')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/save-usersdojos',
+    handler: handlers.actHandler('save_usersdojos')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/remove-usersdojos/{userId}/{dojoId}',
+    handler: handlers.actHandler('remove_usersdojos', ['userId', 'dojoId'])
+  }, {
+    method: 'GET',
+    path: options.basePath + '/dojos/user-permissions',
+    handler: handlers.actHandler('get_user_permissions')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/dojos/user-types',
+    handler: handlers.actHandler('get_user_types')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/update-founder',
+    handler: handlers.actHandlerNeedsUser('update_founder')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/dojos/search-nearest-dojos',
+    handler: handlers.actHandler('search_nearest_dojos')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/countries',
+    handler: handlers.actHandler('list_countries'),
+    config: {
+      cache: {
+        expiresIn: cacheTimes.long
+      }
+    }
+  }, {
+    method: 'POST',
+    path: options.basePath + '/countries/places',
+    handler: handlers.actHandler('list_places')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/countries/continents/lat-long',
+    handler: handlers.actHandler('continents_lat_long')
```

Comment:
shouldn't have `expiresIn` as `list_countries` above? It did had in old version of code.